ci/cron/daily-compat.yml

```diff
@@ -161,8 +159,7 @@ jobs:
             bazel run //ledger-service/http-json-perf:http-json-perf-binary-ce -- \
             --scenario=${scenario} \
             --dars=${DAR} \
-            --reports-dir=${OUT} \
-            --jwt=${JWT}
+            --reports-dir=${OUT}
```

Comment:
Tested locally with
```
set -euo pipefail
eval "$(dev-env/bin/dade assist)"
SCENARIOS="\
com.daml.http.perf.scenario.CreateCommand \
com.daml.http.perf.scenario.ExerciseCommand \
com.daml.http.perf.scenario.CreateAndExerciseCommand \
com.daml.http.perf.scenario.AsyncQueryConstantAcs \
com.daml.http.perf.scenario.SyncQueryConstantAcs \
com.daml.http.perf.scenario.SyncQueryNewAcs \
com.daml.http.perf.scenario.SyncQueryVariableAcs \
"
bazel build //docs:quickstart-model
DAR="${PWD}/bazel-bin/docs/quickstart-model.dar"

for scenario in $SCENARIOS; do
  bazel run //ledger-service/http-json-perf:http-json-perf-binary-ce -- \
  --scenario=${scenario} \
  --dars=${DAR}
done
```

to see if the script is running successfully and the output I got looked fine so far (didn't run it completely because the perf tests take that long).